rpcs3/Emu/RSX/VK/VKDMA.cpp

```diff
@@ -279,7 +279,13 @@ namespace vk
 		}
 #else
 		// Anything running on AMDGPU kernel driver will not work due to the check for fd-backed memory allocations
-		const bool allow_host_buffers = (vendor != driver_vendor::AMD && vendor != driver_vendor::RADV);
+		const bool allow_host_buffers = (
```

Comment:
I think this looks much better
```suggestion
#if defined(__FreeBSD__)
		// Requires root privilege on FreeBSD on Intel
		const bool allow_host_buffers = (vendor != driver_vendor::AMD && vendor != driver_vendor::RADV);
#else
		const bool allow_host_buffers = (vendor != driver_vendor::AMD && vendor != driver_vendor::RADV && vendor != driver_vendor::INTEL);
#endif
```

Reply:
Yes just tried to avoid the repetition but ok no pb.